frontend/src/components/eam/components/groups-table/components/action-cell/action-cell.tsx

```diff
@@ -17,6 +17,13 @@ const ActionCell = (
         label="Delete"
         onClick={handleDelete}
       />
+      <IconButton
+        icon={IconName.GEAR}
+        label={'edit'}
+        to={
+          `${AppRoute.EAM_CONFIGURATE_GROUP}/${id}` as AppRoute.EAM_CONFIGURATE_GROUP_$ID
```

Comment:
```suggestion
          `${AppRoute.EAM_CONFIGURATE_GROUP}/${id}`
```

Reply:
it will not work

Reply:
show me an error

Reply:
![image](https://user-images.githubusercontent.com/73034554/160277349-be5b6a8e-a9b0-42be-86d2-e41318adda0d.png)

Reply:
<img width="409" alt="image" src="https://user-images.githubusercontent.com/42296182/160277474-d00a3c54-030a-416b-b437-0e7af91a2281.png">